gradle/libs.versions.toml

```diff
@@ -26,6 +26,7 @@ opentracing = "0.33.0"
 picocli = "4.7.5"
 protobuf = "3.25.2"
 quarkus = "3.7.2"
+quarkusPlugin = "3.7.3"
```

Comment:
Don't you think we should rather distinguish `quarkus` and `quarkusPlatform`? That's where we usually see discrepancies.

Reply:
Those are all platform-deps. But I've renamed `quarkus` to `quarkusPlatform` - so it's `quarkusPlatform` and `quarkusPlugin` and clear that the one refers to the platform and the other only to the plugin.